Specs/Scene/PickSpec.js

```diff
@@ -94,6 +94,37 @@ defineSuite([
         expect(scene).toPickPrimitive(rectangle);
     });
 
+    it('picks a primitive with a modified pick search area', function() {
+        if (FeatureDetection.isInternetExplorer()) {
+            // Workaround IE 11.0.9.  This test fails when all tests are ran without a breakpoint here.
+            return;
+        }
+
+        camera.setView({
+            destination : Rectangle.fromDegrees(-10.0, -10.0, 10.0, 10.0)
+        });
+
+        var rectangle = createRectangle();
+
+        expect(scene).toPickPrimitive(rectangle,7,7,5);
+    });
+
+    it('does not pick a primitive with a modified pick search area', function() {
+       if (FeatureDetection.isInternetExplorer()) {
+            // Workaround IE 11.0.9.  This test fails when all tests are ran without a breakpoint here.
+            return;
+        }
+
+        camera.setView({
+            destination : Rectangle.fromDegrees(-10.0, -10.0, 10.0, 10.0)
+        });
+
+        createRectangle();
+
+        expect(scene).notToPick(0,0,3);
```

Comment:
I'm starting to think this test and the one above can be combined into a single test, with both pick checks at the bottom.

```
it('picks a primitive with a modified pick search area', function() {
    if (FeatureDetection.isInternetExplorer()) {
        // Workaround IE 11.0.9.  This test fails when all tests are ran without a breakpoint here.
        return;
    }

    camera.setView({
        destination : Rectangle.fromDegrees(-10.0, -10.0, 10.0, 10.0)
    });

    var rectangle = createRectangle();

    expect(scene).toPickPrimitive(rectangle, 7, 7, 5);
    expect(scene).notToPick(7, 7, 3);
});
```

I changed the numbers for the `notToPick` test also to (7, 7) for consistency.